B827EBFFFE98E94C.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "eui-b827ebfffe98e94c", 
```

Comment:
The ID should be just `B827EBFFFE98E94C` (without the `eui-` prefix and all uppercase)

Please fix it and then we'll merge. Thanks!